index.js

```diff
@@ -10,11 +10,9 @@ const obj = () => Object.create(null);
 
 // Prevent caching of this module so module.parent is always accurate
 delete require.cache[__filename];
-const parentDir = path.dirname(module.parent.filename);
```

Comment:
By "keep this here", I meant instead of the other place you moved it to. Line 13 becomes:

```js
const parentDir = path.dirname(module.parent.filename || '.');
```

No reason to resolve it in the class when it can be done once on line 13.